src/python_ms_core/core/auth/provider/hosted/hosted_authorizer.py

```diff
@@ -0,0 +1,24 @@
+import requests
+from requests.models import PreparedRequest
+from ...abstracts.authorizer_abstract import AuthorizerAbstract
+from ...models.permission_request import PermissionRequest
+from ....resource_errors import ExceptionHandler
+
+
+class HostedAuthorizer(AuthorizerAbstract):
+    def __init__(self, config=None):
+        self.config = config
+
+    @ExceptionHandler.decorated
+    def has_permission(self, request_params: PermissionRequest):
+        if request_params.permissions and len(request_params.permissions) > 0:
+            url = self.config.auth_url or None
+            if url:
+                req = PreparedRequest()
+                req.prepare_url(url, request_params.get_search_params())
+                response = requests.get(req.url)
+                return response.json()
```

Comment:
I dont think its JSON response. The response is a simple boolean (true/false).

Reply:
@susrisha, that is fine. 
In Node Axios, we receive the response like `response.data`
In Python Requests, we receive the response like `response.json()`